tools/ec2recovery.py

```diff
@@ -0,0 +1,284 @@
+from datetime import datetime, timedelta
+from dateutil import parser
+import boto3
+import optparse
+import logging
+import os
+import sys
+import time
+
+
+def clear_screen():
+    os.system('clear')
+
+
+def generate_parser():
+    parser = optparse.OptionParser()
```

Comment:
minor pref for argparse also in the standard lib, and also allows for subcommands for cli list snapshots vs restore